sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/jdbc/JdbcPropertiesBeanPostProcessorTest.java

```diff
@@ -0,0 +1,6 @@
+package com.azure.spring.cloud.autoconfigure.jdbc;
+
+
+class JdbcPropertiesBeanPostProcessorTest {
+
+}
```

Comment:
Are we expecting more test cases here? We should test the case when the PostgreSQL auth plugin class is not on the classpath.

Reply:
Sure